include/battle/battle_controller_player.h

```diff
@@ -26,7 +26,12 @@ void BattleControllerPlayer_FightInput(BattleSystem *bsys, BattleContext *ctx);
 void BattleControllerPlayer_ItemInput(BattleSystem *bsys, BattleContext *ctx);
 void BattleControllerPlayer_PokemonInput(BattleSystem *bsys, BattleContext *ctx);
 void BattleControllerPlayer_RunInput(BattleSystem *bsys, BattleContext *ctx);
-void BattleControllerPlayer_SafariBallInput(BattleSystem *bsys, BattleContext *ctx);
+void BattleControllerPlayer_SafariThrowBall(BattleSystem *bsys, BattleContext *ctx);
+void BattleControllerPlayer_SafariThrowMud(BattleSystem *bsys, BattleContext *ctx);
+void BattleControllerPlayer_SafariRun(BattleSystem *bsys, BattleContext *ctx);
+void BattleControllerPlayer_SafariWatching(BattleSystem *bsys, BattleContext *ctx);
+void BattleControllerPlayer_CatchingContestThrowBall(BattleSystem *bsys, BattleContext *ctx);
+u32 TryDisobedience(BattleSystem *bsys, BattleContext *ctx, int *script);
```

Comment:
prefix?

Reply:
it's a static function, but still referenced in the asm so it's here (for now)

Reply:
still should probably have a prefix

Reply:
ehhhh